### PR TITLE
NFT-843: Collateral stats

### PR DESCRIPTION
**Pull request by adamgobes**
Screen Shot 2023-02-06 at 3 40 00 PM

### CODE REVIEW

File: components/Controllers/Collateral/Collateral.module.css [hunk 1]
```diff
@@ -19,3 +19,26 @@
...
+
+.stats > div {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.stats > div:first-of-type,
+.stats > div:nth-of-type(3) {
+  background-color: var(--papr-5);
```
Comment (adamgobes) [inline components/Controllers/Collateral/Collateral.module.css:38]: maybe need some custom theming for this?

Comment (cnasc) [reply components/Controllers/Collateral/Collateral.module.css:38]: I wonder if you should just use the table for this. It's not semantic, but would avoid having to replicate the theming and zebra striping in a one-off component

Comment (adamgobes) [reply components/Controllers/Collateral/Collateral.module.css]: used the Table component! thanks cnasc

**Review by cnasc — APPROVED** [review]
LGTM

**Pull request merged**